stdlib/printf.ml

```diff
@@ -22,10 +22,13 @@ let kbprintf k b (Format (fmt, _)) =
   make_printf (fun acc -> bufput_acc b acc; k b) End_of_acc fmt
 let ikfprintf k oc (Format (fmt, _)) =
   make_iprintf k oc fmt
+let ikbprintf k bc (Format (fmt, _)) =
+  make_iprintf k bc fmt
```

Comment:
You can keep the equality ` let ikbprintf = ikfprintf` here, even if we do not expose the equality in the interface.